dev-infrastructure/configurations/dev-operator-roles.bicepparam

```diff
@@ -1,6 +1,32 @@
 using '../templates/dev-operator-roles.bicep'
 
 param roles = [
+  {
```

Comment:
complete the mock-identities.bicep file with the missing permissions

Reply:
for INT we might to go with miguels proposal - https://issues.redhat.com/browse/ARO-10668